cachito/web/migrations/versions/cb6bdbe533cc_add_env_var_types.py

```diff
@@ -0,0 +1,52 @@
+"""Add environment variables types
+
+Revision ID: cb6bdbe533cc
+Revises: 615c19a1cee1
```

Comment:
FYI, one of us will have to modify this depending on if this PR is merged before #181.

Reply:
Ack.